src/compile/inference/file_level_definitions.pegjs

```diff
@@ -168,8 +168,35 @@ UserValueTypeDef = TYPE __ name: Identifier __ IS __ valueType: NonSemicolonSoup
 
 // ==== Using-for directives
 
-IdentifierPathList =
-    head: IdentifierPath __ tail: (__ COMMA __ IdentifierPath __ )* {
+CustomizableOperator =
+    '&'
+    / '|'
+    / '^'
+    / '~'
+    / '+'
+    / '-'
+    / '*'
+    / '/'
+    / '%'
+    / '=='
+    / '!='
+    / '<'
+    / '<='
+    / '>'
+    / '>=';
+
+UsingEntry =
+    name: (IdentifierPath) alias: (__ AS __ CustomizableOperator)? {
+        if (alias === null) {
+            return name;
+        }
+
+        return { name, alias: alias[3] } as SymbolDesc;
```

Comment:
Out of curiousity, why are we using the `SymbolDesc` interface here instead of the `UsingCustomizedOperator` interface?

Reply:
Fixed as part of b683d3c